### DISCUSSION TITLE
Tissue_labeling_upsampled questions

### DISCUSSION THREAD

**Discussion by cesargutierrez7** [simnibs/simnibs#434] [6. Getting started]
Hello Simnibs team,

I have a couple of inquiries. 

1. I am attempting to edit the tissue_labeling_upsampled.nii.gz file that is derived after meshing the T1 image. I took it over to freesurfer to edit using the normal voxel editing. 

Does the tissue_labeling_upsampled file have any special permissions to take into account to edit?

If not, what must I do to fix this so i can edit tissue_labeling_upsampled to then rerun through simnibs?

I am attempting to follow the tutorial but get stuck on this part.

Please let me know.

**Comment by oulap** [comment]
Please see #432
